aws-codeartifact-repository/aws-codeartifact-repository.json

```diff
@@ -0,0 +1,91 @@
+{
+  "typeName": "AWS::CodeArtifact::Repository",
+  "description": "The resource schema to create a CodeArtifact repository.",
+  "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-codeartifact",
+  "properties": {
+    "RepositoryName": {
+      "description": "The name of the repository.",
+      "type": "string",
+      "pattern": "[a-z][a-z0-9\\-]{0,48}[a-z0-9]",
```

Comment:
I believe the pattern for repo name is different:
https://alpha-docs-aws.amazon.com/codeartifact/latest/APIReference/API_CreateRepository.html

Reply:
oh yeah it is, thanks